appveyor.yml

```diff
@@ -1,14 +1,27 @@
+# need to use Ubunut (linux) because the images we use in tests
```

Comment:
let's not run the tests in appveyor at all, then. cool?

Reply:
sounds good, was only doing it because I could easily communicate with the Docker daemon and wanted to prove this could work in CI, I will move stuff over to droneio